src/libs/mverb/MVerb.h

```diff
@@ -107,8 +107,15 @@ class MVerb
         T OneOverSampleFrames = static_cast<T>(1. / sampleFrames);
         T MixDelta	= (Mix - MixSmooth) * OneOverSampleFrames;
         T EarlyLateDelta = (EarlyMix - EarlyLateSmooth) * OneOverSampleFrames;
-        T BandwidthDelta = static_cast<T>((((BandwidthFreq * 18400.) + 100.) - BandwidthSmooth) * OneOverSampleFrames);
-        T DampingDelta = static_cast<T>((((DampingFreq * 18400.) + 100.) - DampingSmooth) * OneOverSampleFrames);
+
+        const T MaxFreq = std::min(SampleRate * 0.41723f, 18400.0f);
+
+        T BandwidthDelta = static_cast<T>((((BandwidthFreq * MaxFreq) + 100.0f) - BandwidthSmooth) *
+                       OneOverSampleFrames);
+
+        T DampingDelta = static_cast<T>((((DampingFreq * MaxFreq) + 100.0f) - DampingSmooth) *
+                       OneOverSampleFrames);
```

Comment:
Nice fix! 

These are recalculated each millsecond, per processing round:
 - `const T MaxFreq = std::min(SampleRate * 0.41723f, 18400.0f);`
 - `((BandwidthFreq * MaxFreq) + 100.0f)` 
 - `((DampingFreq * MaxFreq) + 100.0f)`

Maybe these chunks can be moved into the one-time `setParameter()` phase, and held in member variables?

Reply:
Yeah, good idea. I gave that a go but managed to break it again somehow; probably missed something about the initialisation order, etc. I'll have another look.

Reply:
Okay, now it's working. Not sure what I screwed up on my first attempt... 🤷🏻